lib/create-contentful-api.ts

```diff
@@ -36,7 +36,7 @@ import { EntryQueries } from './types/query/query'
 import { FieldsType } from './types/query/util'
 import normalizeSelect from './utils/normalize-select'
 import resolveCircular from './utils/resolve-circular'
-import validateTimestamp from './utils/validate-timestamp'
+import validateTimestamp, { ValidationError } from './utils/validate-timestamp'
```

Comment:
I suggest to move `ValidationError` out of validate-timestamp.ts into its own file, as it's now used for more than that